sph/include/sph/hydro_ve/momentum_energy_gpu.cu

```diff
@@ -127,7 +131,6 @@ void computeMomentumEnergy(size_t startIndex, size_t endIndex, unsigned ngmax, D
 }
 
 template void computeMomentumEnergy(size_t, size_t, unsigned,
```

Comment:
ci log shows a build failure [here](https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/44692846847247/4264416954089684/-/jobs/3480066509#L183), if you can't see it, here is the err msg:
```
hydro_ve/momentum_energy_gpu.cu(133): error: no instance of function template "sph::cuda::computeMomentumEnergy" matches the specified type
```

I can reproduce the failure on daint, see /scratch/snx3000/piccinal/pr349/0
Maybe line 130 was wrongly deleted ?

Reply:
It was indeed, I'll fix it.